src/experiment.js

```diff
@@ -28,11 +28,11 @@ class Experiment {
   }
 
   run() {
-    utils.log(`running with options: ${JSON.stringify(this.options)}`)
+    utils.log("running...", this.options)
```

Comment:
Can we write `A/B test running...` or `Alephbet running...` or something similar so it is clearer what's going on here?

Reply:
Ah, sorry, I just saw that you always add the `[Alephbet]` prefix. That's fine!

Reply:
did you try it? :)